tests/bin/check_tron_jobs_test.py

```diff
@@ -562,6 +669,7 @@ def test_job_running_but_action_unknown_already(self):
                             '%Y-%m-%d %H:%M:%S',
                             time.localtime(time.time() - 1800),
                         ),
+                    'end_time': None,
```

Comment:
This shouldn't be None because it has succeeded.

Reply:
Oops, nice catch.